test/iidm/NetworkTest.cpp

```diff
@@ -372,12 +373,20 @@ BOOST_AUTO_TEST_CASE(fictitious) {
     BOOST_CHECK(!network.isFictitious());
 }
 
+std::ostream& operator<<(std::ostream& stream, const Country& country) {
+    stream << Enum::toString(country);
+    return stream;
+}
+
 BOOST_AUTO_TEST_CASE(country) {
     Network network = createTestNetwork();
 
     BOOST_CHECK_EQUAL(4, network.getSubstationCount());
     BOOST_CHECK_EQUAL(2, network.getCountryCount());
 
+    const std::set<Country>& countries = { Country::ES, Country::FR };
+    BOOST_CHECK(countries ==  network.getCountries());
```

Comment:
There is an extra space. Is it possible to use BOOST_CHECK_EQUAL?

Reply:
>Is it possible to use BOOST_CHECK_EQUAL?

No it isn't (problem described in #223)